builder.md

```diff
@@ -159,6 +238,7 @@ A builder MUST provide the following variables to template files:
 - `scheme-system` - obtained from the `system` key of the scheme input
 - `scheme-variant` - obtained from the `variant` key of the scheme input
 - `scheme-is-{{variant}}-variant` - dynamic value built from the `variant` key of the scheme file. e.g. `variant: "light"` provides `scheme-is-light-variant` with a value of `true`.
+- `scheme-override-*` - obtained from the key within `overrides` object within the scheme yaml
```

Comment:
Could we have an example here? Would `diff-changes: base0E` resolve `scheme-override-diff-changes` to `base0E` or something else?